src/v0/destinations/salesforce/utils.js

```diff
@@ -169,7 +181,10 @@ const getAccessToken = async ({ destination, metadata }) => {
 const collectAuthorizationInfo = async (event) => {
   let authorizationFlow;
   let authorizationData;
-  if (isDefinedAndNotNull(event.metadata?.secret)) {
+  console.log(JSON.stringify(event));
```

Comment:
Should remove `console.log` before merging this PR

Reply:
removed